io/plugins/eu.esdihumboldt.hale.io.geopackage.test/src/eu/esdihumboldt/hale/io/geopackage/GeopackageInstanceWriterTest.groovy

```diff
@@ -191,7 +190,8 @@ class GeopackageInstanceWriterTest {
 			}
 
 			abc {
-				a(new BigInteger('1' + Long.MAX_VALUE))
+
+				a(new BigInteger(''+Long.MAX_VALUE))
```

Comment:
Ok, but I would at least add a comment here, that it used to work with `'1'  + Long.MAX_VALUE` and it does no longer for an unknown reason.

Reply:
I have added a comment.